rdrf/rdrf/api_urls.py

```diff
@@ -0,0 +1,23 @@
+from django.conf.urls import patterns, url, include
+from rest_framework.routers import DefaultRouter
+from rdrf import api_views
+from rdrf.custom_rest_router import DefaultRouterWithSimpleViews
+
+router = DefaultRouterWithSimpleViews()
+router.register(r'registries', api_views.RegistryList, base_name='registry')
+router.register(r'users', api_views.CustomUserViewSet)
+router.register(r'workinggroups', api_views.WorkingGroupViewSet)
+router.register(r'countries', api_views.ListCountries, base_name='country')
+router.register(r'genes', api_views.LookupGenes, base_name='gene')
+router.register(r'laboratories', api_views.LookupLaboratories, base_name='laboratory')
+router.register(r'registries/(?P<registry_code>\w+)/indexes', api_views.LookupIndex, base_name='index')
+router.register(r'registries/(?P<registry_code>\w+)/clinitians', api_views.ListClinitians, base_name='clinitian')
```

Comment:
Yeah, it was called clinitian previously and I just copied it over.
Both indices and indexes are acceptable (ex. http://grammarist.com/usage/indexes-indices/) but indexes is maybe more common in US so switching to indices.